OpenXMLSDK.Engine/Word/ReportEngine/Renders/CellExtensions.cs

```diff
@@ -70,7 +70,7 @@ public static class CellExtensions
             }
 
             if (!cell.Show)
-                return wordCell;
+                return null;
```

Comment:
What the behaviour with null instead wordCell ?
How table row is rendered with 3 cell instead 4 if 4 cols are defined ?
The first behaviour of Show in cell is to generate an empty cell instead not rendering it in order to be sure that each row contains the right number of cols and avoid word document errors.

To be discussed

Reply:
After investigation, it fixes the case you describe.
With `return wordCell`, the below case creates an invalid document while when returning null the document is correct.
| H1 | H2              | H3
| A   | Show False | A
| A   | A                | A 

Finally, the table created is 
| H1 | H2 | H3
| A   | A
| A   | A   | A